pkg/controllers/jobset_controller_test.go

```diff
@@ -1219,3 +1308,55 @@ func makeJob(args *makeJobArgs) *testutils.JobWrapper {
 		PodAnnotations(annotations)
 	return jobWrapper
 }
+
+func newJobSet(completionTime, failedTime metav1.Time, ttl *int32) *jobset.JobSet {
```

Comment:
please avoid adding such functions and use the wrappers functions from https://github.com/kubernetes-sigs/jobset/blob/main/pkg/util/testing/wrappers.go (please add missing wrappers if any)